logwarn_openqa

```diff
@@ -45,6 +45,8 @@ $logwarn ${options} -m '\[.*:?(debug|info|warn|error)\]' -p ${file} \
     '!websockets:warn\] A message received from unknown worker connection' \
     `# https://progress.opensuse.org/issues/21018` \
     '!warn\] Stopping worker .* immediately' \
+    `# https://progress.opensuse.org/issues/106613` \
+    '!error\] \[.+\] DBIx::Class::Row::update(): Can\'t update OpenQA::Schema::Result::JobLocks=HASH\(.+\): row not found at .+Locks\.pm' \
```

Comment:
Tests are complaining about a syntax error.
You can run `prove test` locally to test your changes.
I would suggest to replace the `\'` with a simple dot `.`.

Reply:
Actually `prove test_logwarn` worked! Thank you for the tip! The phrasing in the `README.md` wasn't very clear to me, so I'm proposing some clarifications in #20.